frontend/playwright/ui/specs/workspace-viewer-role.spec.js

```diff
@@ -0,0 +1,34 @@
+import { test, expect } from "@playwright/test";
+import { WorkspacePage } from "../pages/WorkspacePage";
+import { presenceFixture } from "../../data/workspace/ws-notifications";
+
+test.beforeEach(async ({ page }) => {
+  await WorkspacePage.init(page);
+
+  const workspacePage = new WorkspacePage(page);
+  await workspacePage.setupEmptyFile(page);
+  await WorkspacePage.mockRPC(
+    page,
+    "get-team?id=*",
+    "workspace/get-team-role-viewer.json",
+  );
+
+  await workspacePage.goToWorkspace();
+});
+
+test("User haven't toolbar", async ({ page }) => {
+  await expect(page.getByTestId("toolbar-options")).toBeHidden();
+  await expect(page.getByTitle("DESIGN")).toBeHidden();
+});
+
+test("User haven't edition menu entries", async ({ page }) => {
+  await page.getByTitle("Main menu").click();
+  await page.getByTestId("file").click();
```

Comment:
Same as above, if there's the possibility of a different locator, let's use that.